### ISSUE TITLE
Feature request: Auto-save

### ISSUE THREAD

**Issue by kissge** [lostintangent/gistpad#276]
**Is your feature request related to a problem? Please describe.**
It would be much of help if GistPad provides auto-saving functionality since I use it for taking notes and want it to be saved periodically, without forgetting.

**Describe the solution you'd like**
Auto-saving functionality like VSCode's built-in one. Maybe configurable (like, off by default).

**Describe alternatives you've considered**
I know VSCode itself has such functionality, but it affects every file I edit with VSCode, which is not desirable.

**Additional context**
N/A

**Comment by lostintangent** [comment]
Yep this makes sense. I'll add this enhancement to the next marketplace update (sometime this week). Thanks for the feedback!
